src/seedu/addressbook/AddressBook.java

```diff
@@ -1143,15 +1143,14 @@ private static String getUsageInfoForExitCommand() {
      * ============================
      */
 
-    /**
-     * Removes sign(p/, d/, etc) from parameter string
-     *
-     * @param s  Parameter as a string
-     * @param sign  Parameter sign to be removed
-     * @return  string without the sign
+    /*
+     * Removes prefix from the given fullString if prefix occurs at the start of the string.
+     * @param fullString  Parameter as a string
+     * @param prefix  Parameter prefix to be removed
+     * @return  string without the prefix
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    private static String removePrefix(String fullString, String prefix) {
+        return fullString.replace(prefix, "");
```

Comment:
The method is not abstract enough to only remove the 'Prefix'. Consider revising the method with replaceFirst()